UNRELEASED_CHANGELOG.md

```diff
@@ -68,6 +68,7 @@
 - Fixed bug related to `ScrollHelper` when `MessageListView` is initialised more than once.
 
 ### ⬆️ Improved
+- The search for mentions now includes transliteration, diacritics removal, and ignore typos. To use transliteration, pass the id of the desired alphabets to `DefaultUserLookupHandler` and set it using `MessageInputView.setUserLookupHandler`. It uses https://unicode-org.github.io/icu/userguide/icu4j/ for transliteration
```

Comment:
Let's update the docs and mention that Transliteration is available starting from API 29 and there is a possibility to provide custom implementation